test/sanity/code-smell/mypy.requirements.txt

```diff
@@ -19,4 +19,4 @@ types-requests==2.32.0.20250328
 types-setuptools==80.4.0.20250511
 types-toml==0.10.8.20240310
 typing_extensions==4.13.2
-urllib3==2.4.0
+urllib3==2.5.0
```

Comment:
These requirements are normally updated in bulk when sanity tests are updated.

Is there a reason for updating only this one requirement?

Reply:
opened due to github nagging me about it, but fine with just ignoring those alerts